composeApp/src/commonMain/kotlin/com/github/familyvault/services/FileCabinetService.kt

```diff
@@ -36,71 +46,270 @@ class FileCabinetService(
             managers = managers,
             tag = AppConfig.FILE_CABINET_THREAD_TAG,
             type = FileCabinetThreadType.IMAGES.toString(),
-            name = familyGroupName,
-            referenceStoreId = storeId,
+            name = "$familyGroupName$IMAGES_SUFFIX",
```

Comment:
Najlepiej jakby nie było tutaj nazwy grupy rodzinnej i to że była poprzednio jest błędem.
Dajmy po prostu tutaj np. "FILE_CABINET_IMAGES" i "FILE_CABINET_DOCUMENTS", możesz wykorzystać np. `FileCabinetThreadType`